azure-cpi.html.md.erb

```diff
@@ -115,7 +117,7 @@ Example of 10GB disk:
 ```yaml
 disk_pools:
 - name: default
-  disk_size: 10_240
+  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
```

Comment:
may be lets put it just above as in your example.

Sent from my iPhone

> On Jul 13, 2016, at 7:26 PM, Chou Hu <notifications@github.com> wrote:
> 
> In azure-cpi.html.md.erb:
> 
> > @@ -115,7 +117,7 @@ Example of 10GB disk:
> >  ```yaml
> >  disk_pools:
> >  - name: default
> > -  disk_size: 10_240
> > +  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
> OK. But disk_size is not a property of cloud_properties, should we put it above "Schema for cloud_properties section:"
> 
> ---
> ## <a id="disk-pools"></a> Disk Pools / Disk Types
> 
> * **disk_size** [Integer, required]: Size of the disk in megabytes. On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
> 
> Schema for `cloud_properties` section:
> 
> * **caching** [String, optional]: Type of the disk caching. It can be either `None`, `ReadOnly` or `ReadWrite`. Default is `None`. Only `None` and `ReadOnly` are supported for premium disks.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>